chassis/invokers/src/main/java/com/griddynamics/jagger/invoker/http/v2/JHttpClient.java

```diff
@@ -0,0 +1,25 @@
+package com.griddynamics.jagger.invoker.http.v2;
+
+/**
+ * An object that represents HTTP-client. <br/>
+ * <p>
+ * Implementation classes should provide an implementation
+ * for {@link #execute(JHttpEndpoint endpoint, JHttpQuery query)} method. <br/>
+ *
+ * @author Anton Antonenko
+ * @since 1.2.7
```

Comment:
I see "Fix version: Release 1.2.7" in Jira. If so, it should be changed there, I think